Makefile.am

```diff
@@ -119,7 +119,6 @@ libcoap_include_HEADERS = \
   $(top_srcdir)/include/coap$(LIBCOAP_API_VERSION)/coap_io.h \
   $(top_srcdir)/include/coap$(LIBCOAP_API_VERSION)/coap_mutex.h \
   $(top_srcdir)/include/coap$(LIBCOAP_API_VERSION)/coap_session.h \
-  $(top_srcdir)/include/coap$(LIBCOAP_API_VERSION)/coap_tcp.h \
```

Comment:
`coap_tcp_internal.h` must be included in EXTRA_DIST (that is why the travis build fails).

Reply:
Corrected and code pushed.  I have checked that the code (well coap-rd fails for another reason) builds on Windows.